atom/browser/api/atom_api_app.cc

```diff
@@ -331,10 +330,6 @@ struct Converter<Browser::LoginItemSettings> {
                                    Browser::LoginItemSettings val) {
     mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
     dict.Set("openAtLogin", val.open_at_login);
-    dict.Set("openAsHidden", val.open_as_hidden);
-    dict.Set("restoreState", val.restore_state);
-    dict.Set("wasOpenedAtLogin", val.opened_at_login);
```

Comment:
Removing `wasOpenedAtLogin` is going to affect a bunch of apps: https://github.com/search?q=wasOpenedAtLogin&type=Code

In [Kap](https://github.com/wulkano/kap), we don't show the main window at system login, but instead just put Kap in the menu bar, but when launched normally, we show the main window. Removing this will hurt our UX...

Reply:
+1 @sindresorhus - this seems quite a regression. Do we still have no way of checking (on mas builds) whether the app was launched from login?